engine/gamesys/src/gamesys/scripts/script_http.cpp

```diff
@@ -322,14 +324,19 @@ namespace dmGameSystem
         lua_pop(L, 1);
 
         assert(top == lua_gettop(L));
+
+        return dmExtension::RESULT_OK;
     }
 
-    void ScriptHttpFinalize(const ScriptLibContext& context)
+    static dmExtension::Result ScriptHttpFinalize(dmExtension::Params* params)
     {
         if (g_Service != 0)
         {
             dmHttpService::Delete(g_Service);
             g_Service = 0;
         }
+        return dmExtension::RESULT_OK;
     }
+
+    DM_DECLARE_EXTENSION(ScriptHttp, "ScriptHttp", 0, 0, ScriptHttpInitialize, 0, 0, ScriptHttpFinalize);
```

Comment:
This is the part that gets called at startup in our generated `dmExportedSymbols()` function.